docs/REST API/rest-api.md

```diff
@@ -7447,6 +7447,7 @@ Returns a list of global queries or team queries.
 | merge_inherited | boolean | query | _Available in Fleet Premium_. If `true`, will include global queries in addition to team queries when filtering by `team_id`. (If no `team_id` is provided, this parameter is ignored.) |
 | page                    | integer | query | Page number of the results to fetch. |
 | per_page                | integer | query | Results per page. |
+| compatible_platform     | string | query | Return queries that are compatible with this platform (not a strict compatibility check) |
```

Comment:
Looks good! I think we should include what the supported options are in the description (i.e. "macOS" not "darwin"). Also maybe clarify how we estimate compatibility. Does this look right? 
```suggestion
| compatible_platform     | string | query | Return queries that only reference tables compatible with this platform (not a strict compatibility check). One of: `"macos"`, `"windows"`, `"linux"` (case-insensitive). |
```

Reply:
The frontend uses `platform=darwin` and `platform=chrome` in the URL, can we have those values? Maybe we should also change that to be `compatible_platform` to align with the API? So:

```suggestion
| compatible_platform     | string | query | Return queries that only reference tables compatible with this platform (not a strict compatibility check). One of: `"darwin"`, `"windows"`, `"linux"`, `"chrome"` (case-insensitive). |
```

Reply:
e.g. this is current FE behavior:
<img width="1239" alt="Screenshot 2024-10-25 at 3 49 42 PM" src="https://github.com/user-attachments/assets/4cc81e22-de62-467d-8db0-097e06d27b06">

Reply:
If we can map `"macos"` to `"darwin"` behind the scenes, I think it's friendlier naming (and I know we want to move towards using that; keeping `platform: "darwin"` when we updated the queries API/UI was mostly for backwards compatibility/easy migration iirc)

Reply:
For sure, sounds good!

```suggestion
| compatible_platform     | string | query | Return queries that only reference tables compatible with this platform (not a strict compatibility check). One of: `"macos"`, `"windows"`, `"linux"`, `"chrome"` (case-insensitive). |
```